### ISSUE TITLE
TensorFlow C API vs. TensorRT: "Op type not registered 'TRTEngineOp' in binary running on ..."

### ISSUE THREAD

**Issue by motrek** [tensorflow/tensorflow#44061]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 20.04.1 LTS
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): docker image (binary)
- TensorFlow version (use command below): v2.3.0-rc2-23-gb36436b087 2.3.0
- Python version:
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: 11.0
- GPU model and memory: GTX 1660 Ti, mobile, 6GB

**Describe the current behavior**
- I have a SavedModel that I convert to a TensorRT-optimized model using trt.TrtGraphConverterV2
- I try to load the resulting model via the TensorFlow C API
- My C++ program displays the following error and exits:
  Op type not registered 'TRTEngineOp' in binary running on [docker container]. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) 'tf.contrib.resampler' should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed.'

**Describe the expected behavior**
I expect the model to be loaded and to be able to use it for inference, as I can with a non-TensorRT-optimized model.

**Standalone code to reproduce the issue**
I'm happy to post code but it seems all you have to do is load a model with TF_LoadSessionFromSavedModel (which doesn't exit) and then, in my case, when I call TF_GraphOperationByName, it prints the error and exits.

**Other info / logs**
If you google this error message, a lot of people have encountered this problem over the last 2 years.
Some people were able to solve it with much earlier versions of TensorFlow by calling:
  TF_LoadLibrary("_trt_engine_op.so", status);
But this does not work for me, the docker image doesn't even seem to have this .so file anywhere, and other people online claim the call is unnecessary.
It seems that TensorRT has been folded into TensorFlow version 2, but maybe somebody forgot to put it into the C API libraries?

**Comment by jappoz** [comment]
I have the same exact problem with tf 2.4, cuda 11.2, trt 7... did someone solve it?

**Comment by Saduf2019** [comment]
@jappoz 
For cuda 11.2 please use tf 2.5 or nightly and let us know.

**Comment by jappoz** [comment]
> @jappoz
> For cuda 11.2 please use tf 2.5 or nightly and let us know.

Done. Switching to tf 2.5.0-rc1 did not solve the problem.

**Comment by sachinprasadhs** [comment]
`tf.contrib` module has been deprecated, as the error message suggests, use the OP which does not make use of OP from deprecated tf.contrib.
